README.md

```diff
@@ -1,9 +1,16 @@
+[![codecov](https://codecov.io/gh/ar-io/ant-pilot/graph/badge.svg)](https://codecov.io/gh/ar-io/ant-pilot)
 # Arweave Name Token Proof of Concept
 
 ## Project setup
```

Comment:
Move this down to developer section at the bottom

Reply:
moved